index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+  
+
+    <div class="header">
+        <h1 id="title">Microverse Survey</h1>
+        <p id="description" class="p">Thank you for taking the time to help us improve the platform</p>
+    </div>
```

Comment:
Please use the header tag instead of div because the header tag is semantically correct and has a specific meaning, whereas div is considered for grouping and styling purposes. Kindly not that using the right semantics in web development makes the application more accessible and covers broad audiences.